packages/lesswrong/components/posts/timeframeUtils.js

```diff
@@ -1,4 +1,4 @@
-import moment from 'moment';
+import moment from 'moment-timezone';
```

Comment:
Somewhat confused about what's going on with the timezone stuff. Is that just an extra bug fix you threw in?

Reply:
I have no idea why it's included with this PR. It looks to me like the commits that are causing this are already in devel. I didn't write them.

Reply:
I assume someone did terrible black magic with git.